.github/workflows/tests.yaml

```diff
@@ -1,25 +1,20 @@
-on: [push, pull_request]
+on: [push, pull request]
```

Comment:
CI doesn't seem to work in the PR commit, have the secret been set up already?
Also, I would make sure that the CI job is correct regarding indentations and so on.

Reply:
Addressed in #37 (was missing the underscore in `pull_request`)